api/.env.sample

```diff
@@ -81,3 +81,4 @@ MRAS_SVC_API_KEY=
 
 # Local development only
 DISABLE_NAMEREQUEST_SOLR_UPDATES=1
+LD_SDK_KEY=
```

Comment:
You can fill this in with the actual Dev key because there's no risk in making it public.

Reply:
Or leave it blank like the other keys in this file. I don't care :)